internal/builder/v1beta1/pipeline.go

```diff
@@ -71,6 +71,16 @@ func Pipeline(name string, ops ...PipelineOp) *v1beta1.Pipeline {
 	return p
 }
 
+// PipelineType will add a TypeMeta to the pipeline's definition.
+func PipelineType() PipelineOp {
+	return func(t *v1beta1.Pipeline) {
+		t.TypeMeta = metav1.TypeMeta{
+			Kind:       "Pipeline",
+			APIVersion: "tekton.dev/v1beta1",
+		}
+	}
+}
```

Comment:
This doesn't need to return a `PipelineOp` as it doesn't take any argument. It can *be* a `PipelineOp`, and then used as `builder.PipelineType`.

```go
func PipelineType(t *v1beta1.Pipeline) {
	t.TypeMeta = metav1.TypeMeta{
		Kind:       "Pipeline",
		APIVersion: "tekton.dev/v1beta1",
	}
}
```